tests/features/support/step-runners.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file Nightwatch interactions
+ * @see {@link http://nightwatchjs.org/api#expect-api|Nightwatch Expect assertions}
+ */
+
+const { client } = require('nightwatch-api');
+const { europeanaId } = require('./europeana-identifiers.js');
+const { url } = require('../config/nightwatch.conf.js').test_settings.default.globals;
+
+const pages = {
+  'home page': `${url}/`,
+  'search page': `${url}/search`,
+  'record page': `${url}/record${europeanaId()}`,
+  'first page of results': `${url}/search?query=&page=1`
+};
+
+function qaSelector(qas) {
```

Comment:
Let's give this a more understandable param name.
qaElementNames.
Also have the methods below that currently take a 'qas' param rename their param.

Maybe add some documentation to this function explaining it's responsible for returning the css selector.